sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonExternalTransform.java

```diff
@@ -72,6 +73,9 @@
   private @Nullable Object @NonNull [] argsArray;
   private @Nullable Row providedKwargsRow;
 
+  Map<String, Coder<?>> outputCoders;
+  private static final String PYTHON_MAIN_OUTPUT_KEY = "0";
```

Comment:
I'm wary of hard-coding this--what if it changes? If there is one and only one output, should we be insensitive to the name?

Reply:
I don't think the key we use here matters when using for a single output. Changed to a different string and added a comment.